gnd/src/main/java/com/google/android/gnd/system/StorageManager.java

```diff
@@ -66,6 +67,7 @@ public StorageManager(
    * Requests for selecting a photo from the storage, if necessary permissions are granted.
    * Otherwise, requests for the permissions and then sends out the request.
    */
+  @Cold
```

Comment:
I may have missed this in previous iterations, but it appears this class isn't really necessary; that the photo picker logic should live in `CameraManager`, and the remote storage logic should live in `RemoteStorageManager`. Please correct me if I'm wrong!

Reply:
I don't agree that this code should be moved to `CameraManager`. For photo picker, we don't need camera permissions or launch camera app. Only READ_EXTERNAL_STORAGE is needed and an intent is started to open any supported gallery/file manager app.

Refactored the changes related to `RemoteStorageManager` as suggested.